### PR TITLE
chore: rename master references to HEAD

### PR DESCRIPTION
**Pull request by sendilkumarn**
Part of #711; 

`/master/HEAD/s`

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/regular-member-pull-request.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ about: You'd like to participate in the Cross Project Council's work as a Regula
 We are thrilled that you'd like to participate in our work in a meaningful way as a Regular Member of the Cross Project Council.
 
 Requirements and rules around becoming a Regular Member can be found in our Governance:
-https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#approving-and-onboarding-regular-members
+https://github.com/openjs-foundation/cross-project-council/blob/main/GOVERNANCE.md#approving-and-onboarding-regular-members
```
Comment (ljharb) [inline .github/PULL_REQUEST_TEMPLATE/regular-member-pull-request.md:10]:
```suggestion
https://github.com/openjs-foundation/cross-project-council/blob/HEAD/GOVERNANCE.md#approving-and-onboarding-regular-members
```
same throughout; this way they aren't hardcoded to any particular branch.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE/regular-member-pull-request.md (hunk 1 above)
Comment (sendilkumarn) [reply .github/PULL_REQUEST_TEMPLATE/regular-member-pull-request.md:10]: That is a good idea. On it 👍🏽

**Review by eemeli — APPROVED** [review]

**Review by ljharb — APPROVED** [review]
👤

**Review by tobie — APPROVED** [review]
Thanks for doing this!

**Review by codebytere — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by joesepi — APPROVED** [review]
Thanks!

**Pull request merged**